utils/tests_fetcher.py

```diff
@@ -195,12 +195,22 @@ def get_test_dependencies(test_fname):
         content = f.read()
 
     # Tests only have relative imports for other test files
-    relative_imports = re.findall(r"from\s+\.(\S+)\s+import\s+([^\n]+)\n", content)
+    # TODO Sylvain: handle relative imports cleanly
+    relative_imports = re.findall(r"from\s+(\.\S+)\s+import\s+([^\n]+)\n", content)
     relative_imports = [test for test, imp in relative_imports if "# tests_ignore" not in imp]
-    dependencies = [os.path.join("tests", f"{test}.py") for test in relative_imports]
-    # Some tests use docstrings with relative imports in them and this could catch them, so we check the files
-    # actually exist
-    return [f for f in dependencies if os.path.isfile(f)]
+
+    # Removes the double trailing '..' for parent imports, and creates an absolute path from the root dir with
+    # `tests` as a prefix.
+    parent_imports = [imp.strip(".") for imp in relative_imports if ".." in imp]
+    parent_imports = [os.path.join("tests", f"{test.replace('.', '/')}.py") for test in parent_imports]
```

Comment:
```suggestion
    parent_imports = [os.path.join("tests", f"{test.replace('.', os.path.sep)}.py") for test in parent_imports]
```
'/' won't work on windows.